auto-dark-light@gihaume/README.md

```diff
@@ -0,0 +1,44 @@
+# Automatic dark/light themes
+
+*Automatic dark/light themes* is Cinnamon applet to automatically switch between dark and light themes at twilight times based on a location.
+
+## Features
+
+- Dump system themes settings in one click to take advantage of the Cinnamon `Themes` settings menu.
+- Sync location from the system `Region` and `City` settings using a local database to determine the geographical coordinates.
+- Enter manually any geographical coordinates if needed.
+- Always sync instantaneously with external changes of prefered color scheme change, region/city, time (in e.g. from a sleep wakeup).
+- Fully event based (no polling).
+- Automatic mode switch can be disabled.
+- Dark/light mode can be switched manually.
+
+## Applet icons legend
+
+<div style="display: flex; align-items: center;">
+    <img src="files/auto-dark-light@gihaume/icons/auto-symbolic.svg" alt="Applet Icon" width="75" height="75" style="margin-right: 10px;">
```

Comment:
You will likely want to update these URLs after this is merged. (You can get ahead of this by looking at other applets and using what will be the eventual path.)